addons/sale/models/sales_team.py

```diff
@@ -80,7 +80,7 @@ def _compute_invoiced(self):
             FROM account_move move
             JOIN account_move_line line ON line.move_id = move.id
             JOIN account_account account ON account.id = line.account_id
-            WHERE move.move_type IN ('out_invoice', 'out_refund', 'in_invoice', 'in_refund')
+            WHERE move.move_type IN ('out_invoice', 'out_refund')
```

Comment:
out_receipt invoices are missing and there is no condition on `exclude_from_invoice_tab` to process only invoice lines. If you need the <sum amount_untaxed (out_invoice/out_receipt)> - <sum amount_untaxed (out_refund)>, you could simply do the following I think:
```
SELECT
                move.team_id         AS team_id,
                SUM(move.amount_untaxed_signed)   AS amount_untaxed_signed
            FROM account_move move
            WHERE move.move_type IN ('out_invoice', 'out_refund', 'out_receipt')
            AND move.payment_state IN ('in_payment', 'paid', 'reversed')
            AND move.state = 'posted'
            AND move.team_id IN %s
            AND move.date BETWEEN %s AND %s
            GROUP BY move.team_id
```

Also plz add a test.